src/psyclone/parse/kernel.py

```diff
@@ -624,29 +633,39 @@ def get_kernel_metadata(name, ast):
         raise ParseError("Kernel type {0} does not exist".format(name))
     return ktype
 
+
 def get_kernel_interface(ast):
     '''Takes the kernel module parse tree and returns the interface part
     of the parse tree.
 
     :param ast: parse tree of the kernel module code
     :type ast: :py:class:`fparser.one.block_statements.BeginSource`
 
-    :returns: Name 'name' of the parse tree of the interface 
-    :rtype: :str `fparser.one.block_statements.Interface.name`
+    :returns: Name of the parse tree of the interface block and
+              the names of the module procedures. Or None, None if
+              there is no interface or the interface has no nodule procedures.
+    :rtype: : `fparser.one.block_statements.Interface.name`,
+              `fparser.one.block_statements.Interface.a.module_procedures` '''
```

Comment:
As noted below, I think (unless I've missed something) that the second return value should just be a list of `str`. Possibly the first return value should also just be `str`?

Reply:
Please could you also document the error that this routine can raise. Our coding standard also mandates a blank line before the closing "'''" characters of the docstring.

Reply:
Done.